src/collective/cover/events.py

```diff
@@ -4,9 +4,8 @@
 from plone import api
 from plone.app.iterate.interfaces import ICheckinEvent
 from plone.app.linkintegrity.handlers import getObjectsFromLinks
-from plone.app.linkintegrity.handlers import referencedRelationship
```

Comment:
@cewing it would be better to have a conditional import depending on Plone version